addons/packages/antrea/1.2.3/test/Makefile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+.DEFAULT_GOAL:=help
+
+help: ## Display this help message
+	# Inspired by Cluster-API Makefile
+	# Any target that has '## ' append to it will be included in the help message
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[0-9A-Za-z_-]+:.*?##/ { printf "  \033[36m%-45s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+lint: ## Run Go code linting
+ifeq ($(origin GOLANGCI_LINT),undefined)
+	@echo "Error! GOLANGCI_LINT env var not set"
+else
+	$(GOLANGCI_LINT) run -v --timeout=5m
+endif
+
+get-deps: ## Get all dependencies
+	go mod download
+
+test: ## Run unit testing suite
+	ginkgo -v .
+
+e2e-test: ## Run e2e testing suite
```

Comment:
can we get ride of the targets we don't need?

Reply:
A standard subset of make tasks was added to all of the modules in https://github.com/vmware-tanzu/community-edition/pull/1143 and it sounds like the expectation is that they are present.

Reply:
it's an odd default but fair enough as a standard.